Intersect.Client.Framework/Plugins/Interfaces/IClientLifecycleHelper.cs

```diff
@@ -28,13 +29,16 @@ public class GameUpdateArgs : EventArgs
 
         public IEntity Player { get; }
 
-        public Dictionary<Guid, IEntity> KnownEntities { get; }
+        public IReadOnlyDictionary<Guid, IEntity> KnownEntities { get; }
 
-        public GameUpdateArgs(GameStates state, IEntity player, Dictionary<Guid, IEntity> knownEntities)
+        public long TimeMs { get; }
```

Comment:
Can you just call this `Delta` and add a doc comment saying it's in milliseconds?

I would prefer we reserve the word "time" for something more absolute-ish, like real-world or time-since-start.

Reply:
Side-note: If you wanted to, you could create a single parent args class that defines the delta, and make both of the other argument classes inherit from it.

Rationale: Both args classes we will always want to provide a certain set of information, like timing, and we won't do it haphazardly by giving one more information than the other.